src/pages/calculator-no-pets-allowed.js

```diff
@@ -186,6 +186,23 @@ const CalculatorNoPetsAllowed = () => {
     sec6Desc,
   } = useWithNoPetCalculatorData()
 
+  const checkEmptyValues = data => {
+    const emptyKeys = [];
+  
+    Object.entries(data).forEach(([key, value]) => {
+      if (!value) {
+        emptyKeys.push(key);
+      }
+    });
+  
+    return emptyKeys;
+  };
+  
+  const emptyConstants = checkEmptyValues(useWithNoPetCalculatorData());
```

Comment:
why do we need this if is not used?

Reply:
I used to check the empty vars. I removed it now.